Source/Scene/Model.js

```diff
@@ -4390,8 +4539,23 @@ define([
                 updateColor(this, frameState, false);
                 updateSilhouette(this, frameState, false);
             }
+        }
+
+        if (this._shouldUpdateSpecularMapAtlas) {
```

Comment:
Can we still take advantage of the faster WebGL 2 path? Or is using an atlas fast enough?

Reply:
This is fast enough and reduces the amount of memory used.